src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.cs

```diff
@@ -2,47 +2,231 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Internal.LowLevelLinq;
+
 namespace System.Threading
 {
     internal static partial class ClrThreadPool
     {
         private const int CpuUtilizationHigh = 95;
         private const int CpuUtilizationLow = 80;
         private static int s_cpuUtilization = 85; // TODO: Add calculation for CPU utilization
-        
-        private static int s_minThreads; // TODO: Initialize
-        private static int s_maxThreads; // TODO: Initialize
 
-        // TODO: SetMinThreads and SetMaxThreads need to be synchronized with a lock
-        // TODO: Compare with CoreCLR implementation and ensure this has the same guarantees.
-        public static bool SetMinThreads(int threads)
+        private static short s_forcedMinWorkerThreads = 0; // TODO: Config
+        private static short s_forcedMaxWorkerThreads = 0; // TODO: Config
```

Comment:
The original code treats the config values as unsigned. This adjustment can be made when loading the config. Please add that to the TODO. Actually now that I think about it, the max adjustment (making sure the value is less than MaxPossibleThreadCount) can also be done when loading the config value.